test/integration/export-404-html/.gitignore

```diff
@@ -0,0 +1 @@
+.next-dev
```

Comment:
seems we didn't need this ignore in this case?

Reply:
Fixed in [`08214a9` (#36910)](https://github.com/vercel/next.js/pull/36910/commits/08214a97e986d939737119ada7582a221cfb8b79)